sdk/servicebus/azure-servicebus/azure/servicebus/_base_handler.py

```diff
@@ -90,29 +108,27 @@ def _generate_sas_token(uri, policy, key, expiry=None):
     return _AccessToken(token=token, expires_on=abs_expiry)
```

Comment:
if we're importing `AccessToken` from azure.core, I think we could remove `_AcessToken` defined by us in the `_base_handler.py` and replace all `_AccessToken` occurrences with `AccessToken` from azure.core.

Reply:
this is an internal change, feel free to push it to next sprint.

Reply:
I considered that but was limiting the scope of this change considering how hot it's coming in.  You OK if we add that as an up-for-grabs issue since it's nicely well defined and self contained? (Made issue [here](https://github.com/Azure/azure-sdk-for-python/issues/13638))

Reply:
sounds good!